src/clean.c

```diff
@@ -30,16 +30,15 @@ static void print_help()
 {
 	fprintf(stderr,
 		"Usage:\n"
-		"	swupd clean [options]\n"
+		"	swupd clean [OPTION...]\n\n"
 		"\n"
 		"Remove cached content used for updates from state directory.\n"
 		"\n"
 		"Options:\n"
 		"   --all                   Remove all the content including recent metadata\n"
 		"   --dry-run               Just print files that would be removed\n"
-		"   -S, --statedir DIR      Specify alternate swupd state directory\n"
-		"   -h, --help              Display this help message\n"
 		"\n");
```

Comment:
isn't this \n going to cause the options to be separated by an empty line? Like this?

```
--all                   Remove all the content including recent metadata
--dry-run               Just print files that would be removed

-S, --statedir DIR      Specify alternate swupd state directory
-h, --help              Display this help message
```